src/js/tracks/text-track-display.js

```diff
@@ -103,7 +121,20 @@ class TextTrackDisplay extends Component {
       cues.push(track['activeCues'][i]);
     }
 
-    window['WebVTT']['processCues'](window, track['activeCues'], this.el_);
+    if (track['kind'] === 'descriptions') {
+
+      if ((overrides.descriptionsPlayback) && (overrides.descriptionsPlayback === 'screenReaderOnly')) {
```

Comment:
I think we should split "showing multiple text tracks at once" out as an issue separate from descriptions. It looks like Chrome natively allows simultaneous display of a subtitles track and a captions track, and that video.js doesn't match that behavior for vtt.js subtitles/captions.